modules/nf-core/glimpse2/chunk/meta.yml

```diff
@@ -0,0 +1,75 @@
+name: "glimpse2_chunk"
+description: Defines chunks where to run imputation
+keywords:
+  - chunk
+  - imputation
+tools:
+  - "glimpse2":
+      description: "GLIMPSE2 is a phasing and imputation method for large-scale low-coverage sequencing studies."
+      homepage: "https://odelaneau.github.io/GLIMPSE"
+      documentation: "https://odelaneau.github.io/GLIMPSE/commands.html"
+      tool_dev_url: "https://github.com/odelaneau/GLIMPSE"
+      doi: "10.1038/s41588-020-00756-0"
+      licence: "['MIT']"
+
+requirements:
+  - AVX2
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - input:
+      type: file
+      description: |
+        Target dataset in VCF/BCF format defined at all variable positions.
+        The file could possibly be without GT field (for efficiency reasons a file containing only the positions is recommended).
+      pattern: "*.{vcf,bcf,vcf.gz,bcf.gz}"
+
+  - input_index:
+      type: file
+      description: Index file of the input VCF/BCF file containing genotype likelihoods.
+      pattern: "*.{vcf.gz.csi,bcf.gz.csi}"
+
+  - region:
+      type: string
+      description: |
+        Target region, usually a full chromosome (e.g. chr20:1000000-2000000 or chr20).
+        For chrX, please treat PAR and non-PAR regions as different choromosome in order to avoid mixing ploidy.
+
+  - map:
+      type: file
+      description: File containing the genetic map.
+      pattern: "*.gmap"
+
+  - model:
+      type: string
+      description: |
+        Algorithm model to use :
+        "recursive": Recursive algorithm
+        "sequential": Recommended sequential algorithm
+        "uniform-number-variants": Experimental. Uniform the number of variants in the sequential algorithm
```

Comment:
I don't understand this.

Reply:
Their is 3 different algorithm that can be used.
You need to select one of those as mentioned [here](https://odelaneau.github.io/GLIMPSE/docs/documentation/chunk/)
That's the unconventional part of this module.
Do you have a better way of checking this ?